Code/GraphMol/MolOps.cpp

```diff
@@ -451,6 +452,45 @@ void assignRadicals(RWMol &mol) {
   }
 }
 
+void cleanupBadStereo(RWMol &mol) {
+  // remove chiral designations and atropisomer designations that cannot be
```

Comment:
I don't believe we support atriopisomer designations yet.

Reply:
True but that is the next PR coming.  I have changed this commnet to remove the atrop reference, but it will be back